### ISSUE TITLE
Graph class, graphviz dependency

### ISSUE THREAD

**Issue by atz** [sul-dlss/dor-services#348]
This gem has a production dependency on `graphviz` based on:
- `Dor::Workflow::Graph` 
- `Dor::WorkflowDs#graph`

This seems to only useful for interactive human debugging, i.e. **not** a dependency this gem should force on dozens of our downstream consumers.  

Does anybody currently use this?  
- Ben suggests that Argo might have used this previously but no longer does based on the rewrite a couple years ago.
- no test coverage

Lines like:
```ruby
  FILL_COLORS = { 'waiting' => 'white', 'ready' => 'white', 'error' => '#8B0000', 'blocked' => 'white', 'completed' => 'darkgreen', 'unknown' => '#CFCFCF' }.freeze
  TEXT_COLORS = { 'waiting' => 'black', 'ready' => 'black', 'error' => 'white', 'blocked' => '#8B0000', 'completed' => 'white', 'unknown' => 'black' }.freeze
  PATTERNS    = { 'waiting' => 'diagonals', 'ready' => 'filled', 'error' => 'filled', 'blocked' => 'diagonals', 'completed' => 'filled', 'unknown' => 'filled' }.freeze
```
seem WAY too coupled to presentation layer to fit comfortably in dor-services.  

I propose removal.

**Comment by blalbrit** [comment]
I don't have use figures, but I would guess no one uses the Diagram view of the workflow modal in Argo - which is the only thing I can think of that would use this. It currently does not successfully load a modal, but you can see the output here:

https://argo.stanford.edu/items/druid:bb000zn0114/workflows/assemblyWF.svg?repo=dor

I don't know if this is what this code was for - but agree we should remove (and remove the "diagram" option in Argo at the same time)

**Comment by jcoyne** [comment]
Fixed by #379
